operator/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3.7
+
+RUN pip install kubernetes==8.0
```

Comment:
I'd prefer to install from `requirementes.txt` to make local development easier.